<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="widget_menu" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>101</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_menu{
background-color: rgb(255, 255, 255);
}</string>
   </property>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox_time">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#checkBox_time
{color: #808080;
background-color:#ffffff;}

#checkBox_time:hover
{color: #2f8dff;
background-color:#ffffff;}</string>
       </property>
       <property name="text">
        <string>time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_blink">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#checkBox_blink
{color: #808080;
background-color:#ffffff;}

#checkBox_blink:hover
{color: #2f8dff;
background-color:#ffffff;}</string>
       </property>
       <property name="text">
        <string>blink</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_angle">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#checkBox_angle
{color: #808080;
background-color:#ffffff;}

#checkBox_angle:hover
{color: #2f8dff;
background-color:#ffffff;}</string>
       </property>
       <property name="text">
        <string>angle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_distance">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#checkBox_distance
{color: #808080;
background-color:#ffffff;}

#checkBox_distance:hover
{color: #2f8dff;
background-color:#ffffff;}</string>
       </property>
       <property name="text">
        <string>distance</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_sex">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#checkBox_sex
{color: #808080;
background-color:#ffffff;}

#checkBox_sex:hover
{color: #2f8dff;
background-color:#ffffff;}</string>
       </property>
       <property name="text">
        <string>sex</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_age">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#checkBox_age
{color: #808080;
background-color:#ffffff;}

#checkBox_age:hover
{color: #2f8dff;
background-color:#ffffff;}</string>
       </property>
       <property name="text">
        <string>age</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_bright">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#checkBox_bright
{color: #808080;
background-color:#ffffff;}

#checkBox_bright:hover
{color: #2f8dff;
background-color:#ffffff;}</string>
       </property>
       <property name="text">
        <string>bright</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_menu">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>385</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Prompt Medium</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_menu{
color: #808080;
background-color: transparent;
border-radius: 20px;
border:1px solid  #808080;
}

#pushButton_menu:hover{
color: #2f8dff;
border:2px solid  #2f8dff;
}
</string>
   </property>
   <property name="text">
    <string>MENU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Prompt SemiBold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_start
{
color: #808080;
background-color:#ffffff;
border-radius: 20px;
border:1px solid  #808080;
}

#pushButton_start:hover{
color: #2f8dff;
border:1px solid  #2f8dff;

}
</string>
   </property>
   <property name="text">
    <string>START</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>430</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Prompt Medium</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_close{
color: #808080;
background-color: transparent;
border-radius: 20px;
border:1px solid  #808080;
}

#pushButton_close:hover{
color: #2f8dff;
border:2px solid  #2f8dff;
}
</string>
   </property>
   <property name="text">
    <string>QUIT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_menu_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>385</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Prompt Medium</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_menu_2{
color: #2f8dff;
background-color: transparent;
border-radius: 20px;
border:2px solid  #2f8dff;
}


</string>
   </property>
   <property name="text">
    <string>MENU</string>
   </property>
  </widget>
  <zorder>label_image</zorder>
  <zorder>widget_menu</zorder>
  <zorder>pushButton_menu</zorder>
  <zorder>pushButton_start</zorder>
  <zorder>pushButton_close</zorder>
  <zorder>pushButton_menu_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
